src/Apps/Order2/Routes/Checkout/Hooks/useCheckoutTracking.ts

```diff
@@ -2,135 +2,153 @@ import {
       },
-
-      expressCheckoutViewed: ({
-        order,
-        walletType,
-      }: {
-        order: { source: string; mode: string }
-        walletType: string[]
-      }) => {
+      expressCheckoutViewed: ({ walletType }: { walletType: string[] }) => {
         const payload: ExpressCheckoutViewed = {
           action: ActionType.expressCheckoutViewed,
           context_page_owner_type: contextPageOwnerType,
           context_page_owner_id: contextPageOwnerId,
           context_page_owner_slug: contextPageOwnerSlug,
-          flow:
-            order.source === "PARTNER_OFFER"
-              ? "Partner offer"
-              : order.mode === "BUY"
-                ? "Buy now"
-                : "Make offer",
+          flow,
           credit_card_wallet_types: walletType,
         }
 
         trackEvent(payload)
       },
-
-      clickedExpressCheckout: ({
-        source,
-        mode,
+      clickedCancelExpressCheckout: ({
         walletType,
       }: {
-        source: string
-        mode: string
         walletType: string
       }) => {
-        const payload: ClickedExpressCheckout = {
-          action: ActionType.clickedExpressCheckout,
+        const payload: ClickedCancelExpressCheckout = {
+          action: ActionType.clickedCancelExpressCheckout,
           context_page_owner_type: contextPageOwnerType,
           context_page_owner_id: contextPageOwnerId,
-          flow:
-            source === "PARTNER_OFFER"
-              ? "Partner offer"
-              : mode === "BUY"
-                ? "Buy now"
-                : "Make offer",
+          // order_id: contextPageOwnerId, // TODO: Clarify whether we are using order_id
+          flow,
           credit_card_wallet_type: walletType,
         }
 
         trackEvent(payload)
       },
+      submittedOrder: (args: { walletType?: string } = {}) => {
+        const { walletType } = args
+        const expressCheckoutValues = walletType
+          ? { credit_card_wallet_type: walletType }
+          : {}
 
-      clickedCancelExpressCheckout: ({
-        source,
-        mode,
-        walletType,
+        const action = buyOrOfferValue(
+          mode,
+          ActionType.submittedOrder,
+          ActionType.submittedOffer,
+        )
+        const payload: SubmittedOrder | SubmittedOffer = {
+          context_page_owner_type: contextPageOwnerType,
+          order_id: contextPageOwnerId,
+          action,
+          flow,
+          ...expressCheckoutValues,
+        }
+
+        trackEvent(payload)
+      },
+
+      clickedBuyerProtection: () =>
+        order2Tracking.clickedBuyerProtection(ContextModule.ordersCheckout),
+
+      clickedPaymentMethod: ({
+        paymentMethod,
+        amountMinor,
+        currency,
       }: {
-        source: string
-        mode: string
-        walletType: string
+        paymentMethod:
+          | "US_BANK_ACCOUNT"
+          | "CREDIT_CARD"
+          | "WIRE_TRANSFER"
+          | string
```

Comment:
nitpick: should we get the update the type here to be from relay? (not sure if we are using it from relay or not)